app/controllers/application_controller.rb

```diff
@@ -131,8 +131,8 @@ def saml_settings
     end
     @saml_settings = SAML::SettingsService.saml_settings
     # TODO: 'level' should be its own class with proper validation
-    level = LOA::MAPPING.invert[params[:level]&.to_i]
-    @saml_settings.authn_context = level || LOA::MAPPING.invert[1]
+    #level = LOA::MAPPING.invert[params[:level]&.to_i]
+    @saml_settings.authn_context = 'dslogon' #level || LOA::MAPPING.invert[1]
```

Comment:
What's up with this?

Reply:
LOL, that's just for me to test right now without the controllers being in place. Ignore. :)

But it will need to be something similar... instead of ?level=1 I'm proposing the new DSL be ?policy=idme, policy=idme-ficam, policy=mhv, policy=dslogon, policy=mfa or something similar.